tags/views.py

```diff
@@ -68,10 +69,11 @@ def tag_cloud(request):
             sentry_sdk.capture_exception(e)
             tvars.update({"error_text": "The search server could not be reached, please try again later."})
         else:
-            if settings.SEARCH_SOUNDS_FIELD_TAGS in results.facets:
+            solr_tags_facet_fieldname = get_solr_fieldname_for_facet(get_solr_fieldname(settings.SEARCH_SOUNDS_FIELD_TAGS))
```

Comment:
this needs to be reworked!
the idea is that outside the solr-specific code, we always interact with search functions using the Freesound field names, not the solr names. In this case this would be "tags", just like some lines above the facets are defining using `settings.SEARCH_SOUNDS_FIELD_TAGS`. I think there's a bug here, because `results.facets` should be using `settings.SEARCH_SOUNDS_FIELD_TAGS` as key instead of "tagfacet". I have to investigate.